clients/client/src/main/java/com/dremio/nessie/client/http/UriBuilder.java

```diff
@@ -17,11 +17,8 @@
 
 import java.io.UnsupportedEncodingException;
```

Comment:
5ffef6e326c4bcd10661d4546a99402a09bb5829 has been added to projectnessie/nessie#575